src/questions/owner_borrow.yaml

```diff
@@ -240,3 +240,751 @@
+  answerKeys:
+    - D
+  subTopics:
+    - memory-allocation
+  difficultyLevel: low
+  choices:
+    - content: "Ownership safety"
+      key: A
+    - content: "memory validation"
+      key: B
+    - content: "Ownership validation"
+      key: c
+    - content: "Memory safety"
+      key: D
+
+- uuid: a74bdf3f-0591-4635-bc21-fa45842feedf
+  type: SingleChoice
+  content: An unsafe memory may lead to a crash, unexpected output and _____
+  hint: reduce security
+  explanation: "it lead to leakage of values because it points to invalid address"
+  answerKeys:
+    - B
+  subTopics:
+    - memory-allocation
+  difficultyLevel: low
+  choices:
+    - content: "compiler shutdown"
+      key: A
+    - content: "Data leakage"
+      key: B
+    - content: "Memory crashes"
+      key: c
+    - content: "Nonr of the above"
+      key: D
+
+- uuid: 7f8afe28-9836-4b25-a11b-ad614065f0ba
+  type: SingleChoice
+  content: A mutable reference is prefixed with
+  hint: mut and an operand
+  explanation: "none"
+  answerKeys:
+    - A
+  subTopics:
+    - borrowing
+  difficultyLevel: low
+  choices:
+    - content: "&mut"
+      key: A
+    - content: "*mut"
+      key: B
+    - content: "+mut"
+      key: c
+    - content: "&%mut"
+      key: D
+
+- uuid: 3a288b4c-f4e1-4ceb-a397-59fb6ff8e09c
+  type: SingleChoice
+  content: Rust is a ____ programming language, so how values are stored is essential to how the language behaves.
+  hint: Nohint
+  explanation: "it is not an object oriented language"
+  answerKeys:
+    - D
+  subTopics:
+    - memory-allocation
+  difficultyLevel: low
+  choices:
+    - content: "Object"
+      key: A
+    - content: "Model"
+      key: B
+    - content: "Static"
+      key: c
+    - content: "system"
+      key: D
+
+- uuid: 6aec742c-ec38-4c6d-bf00-df9edba36e7f
+  type: SingleChoice
+  content: Rust uses a -_____ to enforce its ownership rules and ensure that programs are memory safe.
+  hint: A checker
+  explanation: "RUst needs an enforcer to verify all rules are followed"
+  answerKeys:
+    - B
+  subTopics:
+    - ownership
+  difficultyLevel: low
+  choices:
+    - content: "data checker"
+      key: A
+    - content: "borrow checker"
+      key: B
+    - content: "Check warner"
+      key: c
+    - content: "Compiler"
+      key: D
+
+- uuid: e9c849c6-3807-4fdc-9c45-e6add01c666a
+  type: SingleChoice
+  content: If a reference is invalid the data can be dropped using a construct called _____.
+  hint: how long a variable exists
+  explanation: "start on variable creation and end on variable destruction"
+  answerKeys:
+    - A
+  subTopics:
+    - borrowing
+  difficultyLevel: low
+  choices:
+    - content: "lifetimes"
+      key: A
+    - content: "runtimes"
+      key: B
+    - content: "borrow checker"
+      key: c
+    - content: "FIFO"
+      key: D
+
+- uuid: e6b050d1-22fe-4375-bdcb-91a90f6c2385
+  type: SingleChoice
+  content: What kind of error does violating an ownership rules gives
```

Comment:
Do we get compilation or panic error?  

Also add an explanation

Reply:
its panic https://www.makeuseof.com/rust-ownership/#:~:text=unless%20it's%20borrowed.-,The%20Ownership%20Rules,variable%20owns%20its%20initialized%20value.